src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java

```diff
@@ -0,0 +1,65 @@
+package ch.fmi;
+
+import java.util.Iterator;
+
+import fiji.plugin.trackmate.Model;
+import fiji.plugin.trackmate.Spot;
+import fiji.plugin.trackmate.SpotCollection;
+import fiji.plugin.trackmate.features.spot.SpotAnalyzer;
+
+public class MaxQualitySpotAnalyzer<T> implements SpotAnalyzer<T> {
```

Comment:
Oh, I was using the ImageJ (SciJava) formatting style so far for my own projects.

Would you like `MaxQualitySpotAnalyzer` to be integrated into core TrackMate or your TrackMate-extras project? Then I am happy to apply whatever style you prefer. :-)